buildenv

```diff
@@ -12,11 +12,13 @@ zopen_check_results()
   pfx="$2"
   chk="$1/$2_check.log"
 
-  failures=$(/bin/find tests/ -name "*.output"  ! -size 0 | wc -l)
+  failuretests=$(/bin/find tests/ -name "*.output"  ! -size 0)
+  failures=$(echo "${failuretests}" | wc -l)
   total=$(/bin/find tests/ -name "*.output" | wc -l)
 
-  # Echo the following information to guage build health
+  # Echo the following information to gauge build health
+  echo "${failuretests}" >"$1/$2_check_failures.log"
   echo "actualFailures:$failures"
   echo "totalTests:$total"
-  echo "expectedFailures:14"
+  echo "expectedFailures:15"
```

Comment:
Curious, on which system do you get 15 failures? On Jenkins, we get 14 and on zoscan2b, I'm getting 12.

Reply:
Zoscan2b

Reply:
We should add grep as a dependency. I noticed grep and egrep are used in the tests. Maybe that is why we're getting differences?

Reply:
Will do

Reply:
still have 15 errors here when adding grep. @IgorTodorovskiIBM  Can you run on zoscan2b to see if it's due to the system or whether it's part of my configuration?

Reply:
this is my failures list:
```
cat log/20230129_213052_check_failures.log
tests/heredoc.output
tests/read.output
tests/redir.output
tests/test.output
tests/glob.output
tests/printf.output
tests/trap.output
tests/varenv.output
tests/vredir.output
tests/exec.output
tests/builtins.output
tests/lastpipe.output
tests/array.output
tests/histexp.output
tests/intl.output
```

Reply:
Igor's list:
```
tests/heredoc.output
tests/read.output
tests/redir.output
tests/test.output
tests/glob.output
tests/printf.output
tests/trap.output
tests/varenv.output
tests/vredir.output
tests/exec.output
tests/builtins.output
tests/array.output
```

So 'lastpipe, intl' are the ones that work for Igor and not me.